servers/src/main/java/tachyon/master/lineage/checkpoint/CheckpointScheduler.java

```diff
@@ -52,9 +57,11 @@ public static CheckpointScheduler createScheduler(TachyonConf conf, LineageStore
    * actions to take and the requirement is already met.
    * </p>
    *
-   * @param store a readonly view of the lineage store
+   * @param lineageStoreView a readonly view of the lineage store
+   * @param fileSystemMasterView a readonly view of the file system master
    * @return a scheduling plan (possibly empty) to checkpoint the lineages, or null if no plan is
    *         feasible
    */
-  CheckpointPlan schedule(LineageStoreView store);
+  CheckpointPlan schedule(LineageStoreView lineageStoreView,
```

Comment:
I see. Maybe change the class to `CheckpointPlanner` is better? Just like `RecomputePlanner`?